manual/src/cmds/runtime-tracing.etex

```diff
@@ -205,6 +234,83 @@ instead, call the program as below to produce the same result:
 OCAML_RUNTIME_EVENTS_START=1 ./example
 \end{verbatim}
 
+\subsubsection{s-runtime-tracing-custom-events}{Tracing custom events}
+
+The following program make use of the \texttt{Runtime_events.User} and
```

Comment:
"make use" has a grammar issue, I would just write "uses the Runtim_events.user ...".